docs/revsets.md

```diff
@@ -320,6 +320,53 @@ given [string pattern](#string-patterns).
 
 * `working_copies()`: The working copy commits across all the workspaces.
 
+??? examples
+
+    Given this history:
+    ```
+    E
+    |
+    | D
+    |/|
+    | o C
+    | |
+    o | B
+    |/
+    o A
+    |
+    o root()
+    ```
+
+    **function** `reachable()`
+
+    `reachable(E, A..)` ⇒ `{E,D,C,B}`
+    `reachable(D, A..)` ⇒ `{E,D,C,B}`
+    `reachable(C, A..)` ⇒ `{E,D,C,B}`
+    `reachable(B, A..)` ⇒ `{E,D,C,B}`
+    `reachable(A, A..)` ⇒ `{}` (empty set)
```

Comment:
I suspect these lines would be concatenated.

In that case, maybe we can use bullet list to save blank lines.

Reply:
I think you're right. I didn't even notice that I didn't follow the pattern from the existing examples. I'll send a follow-up PR changing to bullet lists for all of them.